ts/screens/profile/SecurityScreen.tsx

```diff
@@ -160,51 +172,38 @@ const SecurityScreen = (): React.ReactElement => {
           onPress={requestIdentificationAndResetPin}
           testID="reset-unlock-code"
         />
-        <Divider />
         {isIdPayEnabled && (
           /* Reset IDPay code */
           <>
+            <Divider />
             <ListItemNav
               value={I18n.t("idpay.code.reset.title")}
               accessibilityLabel={I18n.t("idpay.code.reset.title")}
               description={I18n.t("idpay.code.reset.body")}
               onPress={idPayCodeHandler}
               testID="reset-idpay-code"
             />
-            <Divider />
           </>
         )}
-
         {/* Enable/disable biometric recognition */}
-        {isFingerprintAvailable && (
-          <ListItemComponent
-            title={I18n.t("profile.security.list.biometric_recognition.title")}
-            subTitle={I18n.t(
-              "profile.security.list.biometric_recognition.subtitle"
-            )}
-            // if the screen reader is disabled, the user can enable/disable
-            // the biometric recognition only by pressing the switch
-            onSwitchValueChanged={
-              !isScreenReaderEnabled
-                ? () => setBiometricPreference(!isFingerprintEnabled)
-                : undefined
-            }
-            switchValue={isFingerprintEnabled}
-            isLongPressEnabled
-            accessibilityState={{ checked: isFingerprintEnabled }}
-            accessibilityRole="switch"
-            // if the screen reader is enabled, the user can enable/disable
-            // the biometric recognition only by pressing the whole ListItemComponent
-            onPress={
-              isScreenReaderEnabled
-                ? () => setBiometricPreference(!isFingerprintEnabled)
-                : undefined
-            }
```

Comment:
Yes you were right! I lost this part migrating from `ListItemComponent` to `ListItemSwitch`. Since the old implementation allowed to click the entire item, it was possible to handle these two scenarios.  Unfortunately the newer implementation doesn't seem to allow this behavior. In relation to this, I preferred to handle the switch change directly from the switch handler for both scenarios.